### PR TITLE
Add more fields to Course model for Ucas Importer

### PR DESCRIPTION
**Pull request by tvararu**
### Context

These will be used by the prototype Course API.

### Changes proposed in this pull request

- Update Course model
- Create db migration
- Pull in fields using XlsReader
- Pass them along through UcasCourse and Course model

### Guidance to review

`HasBeenPublished` already existed on `UcasCourse` but was not being used as far as I can tell.

### CODE REVIEW

**Comment by tvararu** [comment]
[UCAS course API: import missing data into Manage Courses DB](https://trello.com/c/Gke8lUak/781-ucas-course-api-import-missing-data-into-manage-courses-db)

File: src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs [hunk 1]
```diff
@@ -328,5 +331,14 @@ public List<UcasNoteText> ReadNoteText(string folder)
             _logger.Information(noteTexts.Count + " note texts loaded from xls");
             return noteTexts;
         }
+
+        private int? ParseIntCell(string cellValue)
+        {
+            int? result = null;
+            try {
+                result = int.Parse(cellValue);
```
Comment (defong) [inline src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: TryParse instead please

File: src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public partial class AddMoreUcasFieldsToCourse : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "english",
+                table: "course",
+                nullable: true);
+
+            migrationBuilder.AddColumn<string>(
```
Comment (MartinBentley) [inline src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs]: Shouldn't this be a boolean? Then check for 'Y' or empty in the cell

File: src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs (hunk 1 above)
Comment (tvararu) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: Done

File: src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs (hunk 2 above)
Comment (tvararu) [reply src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs]: Good suggestion

Comment (tvararu) [reply src/ManageCourses.Domain/Migrations/20181212152132_AddMoreUcasFieldsToCourse.cs]: Done

File: src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs [hunk 3]
```diff
@@ -62,10 +63,13 @@ public List<UcasCourse> ReadCourses(string folder, IList<UcasCampus> campuses)
                         Publish = row.GetCell(columnMap["PUBLISH"]).StringCellValue.Trim(),
                         Status = row.GetCell(columnMap["STATUS"]).StringCellValue.Trim(),
                         VacStatus = row.GetCell(columnMap["VAC_STATUS"]).StringCellValue.Trim(),
-                        HasBeenPublished = row.GetCell(columnMap["HAS_BEEN_PUBLISHED"]).StringCellValue.Trim(),
+                        HasBeenPublished = hasBeenPublished.Equals('Y'),
```
Comment (tvararu) [inline src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]:
```csharp
var str1 = "Y";
var str2 = 'Y';
str1.equals(str2); // True or False?
```

🤔

Comment (tvararu) [reply src/ManageCourses.UcasCourseImporter/xls-reader/XlsReader.cs]: screenshot 2018-12-17 at 11 13 28

**Pull request merged**